### DISCUSSION TITLE
Support for eigenvalue uncertainty based adaptive sampling

### DISCUSSION THREAD

**Discussion by hnadeem2** [deeptime-ml/deeptime#288] [Q&A]
Hello!

I was looking to implement the adaptive sampling technique from [here](https://doi.org/10.1063/1.2740261). The idea is to start sampling from states which maximally reduce the variance in eigenvalue of the transition matrix. 
The computation involves calculation of the sensitivity vectors among other quantities. I wanted to ask if we can use Bayesian MSMs in deeptime to achieve this computation, or if there are other methods that can be helpful for this.

Thanks,
Hassan

**Comment by clonker** [comment]
Hi Hassan, I am not familiar with the method but you may be able to use this:
https://deeptime-ml.github.io/latest/api/generated/deeptime.markov.tools.analysis.eigenvector_sensitivity.html#deeptime.markov.tools.analysis.eigenvector_sensitivity
Other than that you can of course use bayesian sampling to estimate the variance around quantities derived from a msm. But I feel like the analytic method above is - if applicable - probably preferable.
Cheers, Moritz.
